utils/capabilities.c

```diff
@@ -36,6 +36,37 @@ gbCapabilitiesEnumParse(const char *cap)
 }
 
 
+bool
+gbTcmurCmdTimeoutDependenciesVersionCheck(void)
+{
+  char *out = NULL;
+  int ret = true;
+
+
+  out = gbRunnerGetOutput(CONFIGSHELL_VERSION);
+  if (!gbDependencyVersionCompare(CONFIGSHELL_SEMICOLON, out)) {
+    LOG ("mgmt", GB_LOG_INFO,
+         "to support 'tcmur_cmd_timeout' the configshell version at least >= %s",
```

Comment:
please fix the grammar the msg is not clear.

Reply:
will fix it.